Libplanet.Tests/TestUtils.cs

```diff
@@ -276,6 +254,49 @@ public static byte[] GetRandomBytes(int size)
             return block;
         }
 
+        public static Block<T> AttachStateRootHash<T>(
+            Block<T> block, IStateStore stateStore, IAction blockAction)
```

Comment:
How about turning this to an extension method? E.g.:

```suggestion
        public static Block<T> AttachStateRootHash<T>(
            this Block<T> block, IStateStore stateStore, IAction blockAction)
```

Or:

```suggestion
        public static Block<T> AttachStateRootHash<T>(
            Block<T> block, this IStateStore stateStore, IAction blockAction)
```

Reply:
It is a very cool idea, but I couldn't because of this compiler message:

> Extension method can only be declared in non-generic, non-nested static class

Reply:
AFAIK every method of `TestUtils` is already `static`.  You can add a `static` keyword to its class declaration.

Reply:
I understood the message as that I can't do that because it is generic method. 👀
It was correct that you said, and I amended yours in 07403de